xbmc/rendering/CMakeLists.txt

```diff
@@ -4,8 +4,10 @@ set(HEADERS RenderSystem.h
             RenderSystemTypes.h)
 
 if(TARGET ${APP_NAME_LC}::OpenGl OR TARGET ${APP_NAME_LC}::OpenGLES)
-  list(APPEND SOURCES MatrixGL.cpp)
-  list(APPEND HEADERS MatrixGL.h)
+  list(APPEND SOURCES GLExtensions.cpp
+                      MatrixGL.cpp)
+  list(APPEND HEADERS GLExtensions.h
+                      MatrixGL.h)
```

Comment:
I go one step further for history and do

```cmake
  list(APPEND HEADERS GLExtensions.h
                      MatrixGL.h
  )
```

or


```cmake
  list(APPEND HEADERS
    GLExtensions.h
    MatrixGL.h
  )
```

but it's your style

Reply:
I decided to keep the style consistent with the other lists in the file.